oidc-controller/api/verificationConfigs/models.py

```diff
@@ -38,11 +39,17 @@ class VerificationProofRequest(BaseModel):
     requested_predicates: list[ReqPred]
 
 
+class MetaData(BaseModel):
+    title: str = Field()
+    claims: list[str] = Field()
+
```

Comment:
Think we'd set claims to be allowable to be missing or empty (in the event there's only a title)
I think I'm getting an error in one of those cases?

![image](https://github.com/user-attachments/assets/dbb363f6-71fa-44b6-9988-918765159f96)

```
1 validation error for VerificationConfig\nmetadata.claims\n  Field required [type=missing, input_value={'title': '123'}, input_type=dict]\n    For further information visit https://errors.pydantic.dev/2.9/v/missing\n", "logger": "api.main", "level": "error", "timestamp": "2024-11-08T23:38:01.766230Z"}
```

Empty array seems all good:
![image](https://github.com/user-attachments/assets/2388e1db-bf42-4b60-b823-bfda26e2f342)

Reply:
Added support for this thanks for the suggestion